src/WebJobs.Script/Host/ScriptHostManager.cs

```diff
@@ -444,6 +444,10 @@ protected virtual void OnInitializeConfig(ScriptHostConfiguration config)
                 // to prevent host startup failure
                 config.HostConfig.DashboardConnectionString = null;
             }
+
+            // If set, will run the SDK in a Blob Container specified by a SAS connection. 
+            config.SaSContainerConnectionString = 
```

Comment:
The resolution code from app setting => config property for the regular connection string lives in the core SDK. I think it should for this new property as well. We shouldn't need any changes in this repo for this.